src/ec/packages/ec-component-select/select.story.js

```diff
@@ -36,11 +36,10 @@ storiesOf('Components/Forms/Select', module)
         optional_text: text('Optional text', '(optional)'),
         width: inputWidthSelect,
         id: 'example-id',
```

Comment:
necessary to set and example id here?

Reply:
Right, but more than that i saw that we were not using the data from the local folder but still the specs, i improved the story file, now it should be fine ;)